Makefile

```diff
@@ -6,11 +6,6 @@ MANAGER_IMAGE_TAG ?= latest
 MANAGER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(MANAGER_IMAGE_NAME):$(MANAGER_IMAGE_TAG)
 MANAGER_IMAGE ?= $(IMAGE_REGISTRY)/$(MANAGER_IMAGE_FULL_NAME)
 
-HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
```

Comment:
we agreed to keep only handler, not manager (because of it sounds centralized)

Reply:
Is PR is draft, no ready for review, my idea is to rename it to handler, it was easier to use it as manager since the name was ar more places.